hoot-core-test/src/test/cpp/hoot/core/scoring/AttributeComparatorTest.cpp

```diff
@@ -32,10 +32,7 @@
 #include <hoot/core/io/OsmXmlReader.h>
 #include <hoot/core/schema/OsmSchema.h>
 #include <hoot/core/scoring/AttributeComparator.h>
-
-#define int64 opencv_broken_int
 #include <hoot/core/util/OpenCv.h>
-#undef int64
```

Comment:
All of the `#undef`s are removed except for the one in `<hoot/core/util/OpenCv.h>`.  They aren't needed all over the place.

Reply:
Good.
These were an artifact from OpenCV/Geos funkyness: https://github.com/opencv/opencv/issues/7573